go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/jessevdk/go-flags v1.4.0
+	github.com/jstemmer/go-junit-report v0.0.0-20190106144839-af01ea7f8024
```

Comment:
these are coming up after #982 on bootstrap, can't be bothered doing a separate PR.